### DISCUSSION TITLE
Is there a way to set orientation and position on Stellarium using a WCS or similar file from a plate solver?

### DISCUSSION THREAD

**Discussion by ksd3** [Stellarium/stellarium#3651] [Q&A]
I'm using astrometry.net to plate solve images taken with phone cameras. I want to check how accurate the output of astrometry.net is compared to what we'd see in real life. Is there a way I can use any of the generated output files and set my position and orientation in stellarium, assuming the time, date, and location of the image are known?

**Comment by 10110111** [comment]
What output files does it generate? What information do they contain?

You can load the photo into Lens Distortion Estimator plugin and position it at the desired location and in the desired orientation in the sky. Note that this plugin is still experimental, so if you try estimating lens distortion with it, this may not work properly (I know some cases where the result is very suboptimal).

**Reply by gzotti** [reply]
Using astrometry.net to add your DSO images is mentioned in User Guide, section 8.2. Stellarium has no direct interface like "load metadata (date/time/GPS location/view direction) from digital image".
